spec/readme.md

```diff
@@ -116,7 +116,13 @@ Note, the `// comments` shown below are not JSON syntax and cannot be included i
 			"type": { "enum": [ "dicom" ] },
 
 			"description": "Any dicom file."
-		}
+		},
+
+		// A contextual key-value, provided by the environment. Used for values that are generally the same for an entire project.
+		// Not guaranteed to be found - the gear should decide if it can continue to run, or exit with an error.
+		"matlab_license_code": {
+			"base": "context",
+		},
```

Comment:
remove trailing comma

Reply:
I actually leave those in intentionally, since the comments make it invalid JSON anyway (I have the code block set to `js` syntax), just so diffs can be cleaner. Maybe that's dumb.